package.json

```diff
@@ -24,18 +24,18 @@
     "assert-diff": "^1.0.1",
     "babel": "^5.8.20",
     "babel-core": "^5.8.20",
-    "babel-eslint": "^4.0.5",
+    "babel-eslint": "^6.0.0",
     "babel-loader": "^5.3.2",
     "codecov.io": "^0.1.6",
     "coveralls": "~2.11.4",
-    "eslint": "^1.2.1",
+    "eslint": "2.7.0",
```

Comment:
Was removing the `^` intentional?

Reply:
Yes. Due to experiencing breaking changes in eslint in the past, I think it's safer to pin to the version we want.